cleanlab_studio/internal/enrichment_utils.py

```diff
@@ -107,11 +107,13 @@ def get_constrain_outputs_match(
 ) -> str:
     """Extracts the provided output values from the response using regex patterns. Return first extracted value if multiple exist.
     If no value out of the possible `constrain_outputs` is directly mentioned in the response, the return value with greatest string similarity to the response is returned (along with a warning).
+    If there are no close matches between the LLM response and any of the possible `constrain_outputs`, then the last entry of the `constrain_outputs` list is returned.
```

Comment:
This seems like confusing behavior to me that makes it hard to decide if the LLM is behaving badly or there are wrong outputs listed in contrain_outputs (think typos/capitalization?). I think the better behavior is to return np.nan and have the user handle this case (unless you've already considered this)

Reply:
Nevermind, I see there is a preference to making the last value the "other".  I think that is ok then